### DISCUSSION TITLE
Zoom To Data is missing Time Series Panel.

### DISCUSSION THREAD

**Discussion by mertbakir** [grafana/grafana#50215] [Dashboards, Panels & Visualizations]
I did a quick search couldn't find a similar topic, sorry if this is discussed before. anyway...

Can we add this superior technology to the Time Series panel. The ss from old Graph panel. 

![image](https://user-images.githubusercontent.com/43188411/172051336-8c79568d-5ab4-4cff-a930-ae085c2ff3f5.png)

With the time series panel. I'm selecting like last 5 years. then manually zooming into the thin line until I'm in the correct time period, which is very irritating.

**Comment by ryantxu** [comment]
In 9x this now works:  
![2022-06-22_20-47-17 (1)](https://user-images.githubusercontent.com/705951/175204795-be7c4045-5bf9-4533-b2b6-945854eddab1.gif)

It might be in 8.5, but I have not checked

**Reply by mertbakir** [reply]
Great!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
